src/boot/O2/loadfragment2.c

```diff
@@ -131,22 +132,27 @@ void Overlay_Relocate(void* allocatedRamAddr, OverlayRelocationSection* ovlReloc
     }
 }
 
-size_t Overlay_Load(uintptr_t vromStart, uintptr_t vromEnd, void* ramStart, void* ramEnd, void* allocatedRamAddr) {
-    uintptr_t vramStart = (uintptr_t)ramStart;
-    uintptr_t vramEnd = (uintptr_t)ramEnd;
+size_t Overlay_Load(uintptr_t vromStart, uintptr_t vromEnd, void* vramStart, void* vramEnd, void* allocatedRamAddr) {
+    s32 pad[2];
     s32 size = vromEnd - vromStart;
     uintptr_t end;
     OverlayRelocationSection* ovlRelocs;
 
-    if (gOverlayLogSeverity >= 3) {}
     if (gOverlayLogSeverity >= 3) {}
 
+    size = vromEnd - vromStart;
     end = (uintptr_t)allocatedRamAddr + size;
+
+    if (gOverlayLogSeverity >= 3) {}
```

Comment:
```suggestion
    //! FAKE:
    if (gOverlayLogSeverity >= 3) {}
```

Reply:
They are real, though knowledge actually comes from the japanese 1.0 version. I'll put the strings in comments since we will need them eventually.